examples/teleop/src/index.js

```diff
@@ -122,11 +94,9 @@ connectWebRTC()
     app.ports.sendBaseSetPower.subscribe(async ({ linear, angular }) => {
       console.log('linear', linear);
       console.log('angular', angular);
-      // TODO: stop requiring users to import commonApi, if possible
-      const linearVec = new commonApi.Vector3();
-      const angularVec = new commonApi.Vector3();
-      linearVec.setY(linear);
-      angularVec.setZ(angular);
+
+      const linearVec = { x: 0, y: linear, z: 0 };
+      const angularVec = { x: 0, y: 0, z: angular };
```

Comment:
not related to any change in this PR, but needed to adjust to fit an api change from an earlier version bump